### DISCUSSION TITLE
[Support]: Full-Screen Kiosked Birdseye CCTV Display Terminal

### DISCUSSION THREAD

**Discussion by dolai1** [blakeblackshear/frigate#10413] [General Support]
### Describe the problem you are having

Admittedly, this is not purely a Frigate question, but it seems like the best place to ask it:
How can I get a dedicated terminal to display the JSMPEG stream in full-screen on boot?

The system is a dedicated Raspberry Pi 5 running Bookworm (64-bit), which means Wayland/Wayfire. Chromium is the current browser, but I'm willing to use another if it would work.
I was able to get this to work with VLC and the RTSP stream, but this is unstable (dropped frames maybe once a minute and crashes within a few hours, necessitating intervention to bring the view back up, which is unacceptable in this application; it needs to stay up indefinitely without intervention).
The JSMPEG steam seems much more stable (if perhaps with slightly lower frame-rate and a little more laggy), but I can only get the Birdseye page up with the embedded video stream, not a full-screen view of the stream as I would have by clicking on the stream.

Wayland doesn't allow spoofing of keyboard and mouse inputs without root access and I wasn't able to get ydotool to work (xdotool would be the way to go on a system running X11).
It seems like the best would be to have an address for the browser to open directly, but I don't know what that would be (just using /birdseye for now).

Your input would be appreciated... thank you.

### Version

0.12.1

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

PCIe

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment]
See https://github.com/blakeblackshear/frigate/issues/1891

**Reply by dolai1** [reply]
Ah, so it's a feature request at this stage?

**Reply by NickM-27** [reply]
yes

**Reply by dolai1** [reply]
Alright; thanks.

Any suggestions in the meantime, aside from using the RTSP stream, or workaround of displaying a HA card (which I had not thought of)?

**Comment by adamgranted** [comment]
@dolai1 I ran into the same thing and put together a container for this specifically. You can also run on bare metal if you want.
https://github.com/adamgranted/rtspstreamer

**Comment by dolai1** [comment]
@adamgranted thanks; interesting. I adjusted my approach a little bit and added some flags from your "bare metal" version.

I am now using VLC and the RTSP stream. "~/.config/lxsession/LXDE-pi/autostart" is:
```
#@lxpanel --profile LXDE-pi
@xset s off
@xset -dpms
@xset s noblank
@vlc -I dummy rtsp://FRIGATEIP:8554/birdseye --fullscreen --no-video-title-show --no-osd --no-audio --network-caching=1000 --rtsp-tcp --codec avcodec --avcodec-hw=none --file-caching=1000 --live-caching=1000 --realrtsp-caching=1000 --loop --quiet
```
